File: android/aspectj/AspectJ_2017_6_11/app/src/main/java/top/goluck/aspectj_2017_6_11/BaseApplication.java
package top.goluck.aspectj_2017_6_11;

import android.app.Application;
import top.goluck.test.SdkManager;

/**
 * 作者：luck on 2017/6/13 15:14
 * 邮箱：fc_dream@163.com
 * AspectJ_2017_6_11
 */
public class BaseApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        SdkManager.init(this);
    }
}

File: android/kotlin/Retrofit_OkHttp_RxJava_2017_7_30/app/src/main/java/top/goluck/retrofit_okhttp_rxjava_2017_7_30/base/BaseApplication.kt
package top.goluck.retrofit_okhttp_rxjava_2017_7_30.base

import android.app.Application

/**
 * Created by luck on 2017/7/30.
 */
class BaseApplication : Application(){

    override fun onCreate() {
        super.onCreate()
        context = this
    }


    companion object {
        var context: BaseApplication? = null
            private set
    }
}